Formula/kmc.rb

```diff
@@ -5,15 +5,17 @@ class Kmc < Formula
   url "https://github.com/marekkokot/KMC/archive/v3.0.1.tar.gz"
   sha256 "0dbc9254f95541a060d94076d2aa03bb57eb2da114895848f65af0db1e4f8b67"
   head "https://github.com/marekkokot/KMC.git"
+  revision 1
 
   bottle do
     root_url "https://linuxbrew.bintray.com/bottles-bio"
     cellar :any_skip_relocation
     sha256 "3f6989a7acf4d982edd19b96360ece099f288f919c74f4dbf5cd5a59bf295fd9" => :x86_64_linux
   end
 
-  # fatal error: 123:10: 'ext/algorithm' file not found
-  depends_on :linux
+  fails_with :clang # fatal error: 123:10: 'ext/algorithm' file not found
+  
+  depends_on "gcc" if OS.mac?
```

Comment:
This line may not be needed. It depends on whether `kmc` links against any libraries provided by `gcc`. `brew linkage kmc` will tell you. If it doesn't need one of OpenMP, C++, or Fortran, it may not need `gcc` at run time.

```
❯❯❯ brew linkage kmc                       master ✱ ◼
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libbz2.1.0.dylib
  /usr/lib/libz.1.dylib
```
Looks like no `gcc` linkage.

Reply:
See https://github.com/brewsci/homebrew-bio/pull/167